xgcm/test/test_grid.py

```diff
@@ -786,18 +786,19 @@ def test_boundary_kwarg_same_as_grid_constructor_kwarg():
     "metric_axes,metric_name",
     [("X", "dx_t"), ("Y", "dy_ne"), (["Y", "X"], "dy_n")],
 )
-def test_interp_like(metric_axes, metric_name):
+@pytest.mark.parametrize("periodic", [True, False])
+def test_interp_like(metric_axes, metric_name, periodic):
 
     ds, coords, _ = datasets_grid_metric("C")
-    grid = Grid(ds, coords=coords)
+    grid = Grid(ds, coords=coords, periodic=periodic)
     grid.set_metrics(metric_axes, metric_name)
     metric_available = grid._metrics.get(frozenset(metric_axes), None)
     metric_available = metric_available[0]
     interp_metric = grid.interp_like(metric_available, ds.u)
-    test_metric = grid.interp(ds[metric_name], metric_axes)
+    expected_metric = grid.interp(ds[metric_name], metric_axes)
 
-    xr.testing.assert_equal(interp_metric, test_metric)
-    xr.testing.assert_allclose(interp_metric, test_metric)
+    if xr.testing.assert_equal(interp_metric, expected_metric) is False:
```

Comment:
assert works by raising an error so this won't work.

I would just keep the `allclose` version; it's the right one for floating point comparisons

Reply:
If you really wanted to do that comparison, do it this way:
```
if interp_metric.equals(expected_metric) is False:
```

but for here, `assert_allclose` should be good enough